node/src/components/rpc_server/rpcs/state.rs

```diff
@@ -470,7 +470,7 @@ impl RpcWithOptionalParams for GetAuctionInfo {
 
 /// Identifier of an account.
 #[derive(Serialize, Deserialize, Debug, Clone, JsonSchema)]
-#[serde(deny_unknown_fields, rename_all = "snake_case")]
+#[serde(deny_unknown_fields, untagged)]
```

Comment:
We could extend the API to allow a message like this:
```
{
  "jsonrpc": "2.0",
  "method": "state_get_account_info",
  "params": {
    "account_identifier": "account-hash-c029c14904b870e64c1d443d428c606740e82f341bea0f8542ca6494cef1383e",
    "block_identifier": null
  },
  "id": 2475579477138713091
}
```
by renaming the `public_key` field to `account_identifier` in the `GetAccountInfoParams` struct and adding `#[serde(alias = "public_key")]` to the new `account_identifier` field. if we feel the clarity added is worthwhile.

Reply:
Good idea!  Unfortunately the aliasing isn't visible in the generated schema (it would show only `account_identifier`) but I doubt this is a problem.  I'd be happy for you to add this change to the PR.